src/main/java/org/tridiots/tdfs/NameNode.java

```diff
@@ -27,12 +29,12 @@ public void stop() {
 
     @Override
     public String sendHeartBeat(String sender) {
-        System.out.println("get heartbeat from " + sender);
+        logger.info("get heartbeat from " + sender);
```

Comment:
we can use it like `logger.info("get heartbeat from {}", sender);`. Also other places.